### PR TITLE
[full-ci] More app top bar fixes

### PR DESCRIPTION
**Pull request by dschmidt**
## Related Issue

- Fixes <issue_link>

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-pkg/src/composables/appDefaults/useAppFileHandling.ts [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import { unref } from '@vue/composition-api'
 import qs from 'qs'
 
-import { Resource } from 'files/src/helpers/resource'
+import { Resource } from 'web-client'
 import { MaybeRef } from '../../utils'
 import { ClientService } from '../../services'
 import { DavProperties } from '../../constants'
+import { buildResource } from 'files/src/helpers/resources'
```
Comment (dschmidt) [inline packages/web-pkg/src/composables/appDefaults/useAppFileHandling.ts:8]: This should not stay in `files`, but where do we want this and a bunch of other helpers? in `web-pkg` or `web-client`?
Let's discuss tomorrow...

**Comment by ownclouders** [comment]
Results for <strong>oC10IntegrationApp2</strong> https://drone.owncloud.com/owncloud/web/27099/73/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oC10IntegrationApp1</strong> https://drone.owncloud.com/owncloud/web/27099/72/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Review by kulmann — APPROVED** [review]
Love it! 🤩

**Pull request merged**